src/model/reference/index.ts

```diff
@@ -86,3 +86,9 @@ export class Reference extends BackmatterEntity {
     return new refInfoClass(data);
   }
 }
+
+export function createReferencesState(referencesXml: Element[]): Reference[] {
```

Comment:
Need a unit test for empty array and array of Elements (can probably copy from one of the other manuscript property state factories).